nlubridge/nlu_dataset.py

```diff
@@ -279,19 +308,17 @@ def clip_by_intent_frequency(self, max_frequency, min_frequency=None):
                 'internet_langsam': 700,
                 'komplettausfall_eindeutig': 700})
 
+
         :param max_frequency: maximum number of samples per intent to
             include in the returned dataset
-        :type max_frequency: int
         :param min_frequency: maximum number of samples per intent to
```

Comment:
Typo, muss heißen: 
:param min_frequency: minimum number of samples per intent to

Reply:
Korrigiert